docker-compose.override.yaml

```diff
@@ -12,3 +12,4 @@ services:
       - ./.flake8:/home/python/app/.flake8
       - ./visiont:/home/python/app/visiont
       - /tmp:/data
+      - /nas:/nas
```

Comment:
This and the line above are a bit ugly, because it will only work on our machines :see_no_evil: